src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -194,6 +201,10 @@ public function denormalize($data, $class, $format = null, array $context = arra
             }
         }
 
+        if (!empty($extraAttributes)) {
+            throw new ExtraAttributesException(sprintf('Extra attributes are not allowed ("%s" are unknown).', implode('", "', $extraAttributes)));
```

Comment:
I don't really like doing that. And it's not the current behavior for serializer exceptions (like the ``CircularReferenceException``).

However, I pushed this change.